bitrise.yml

```diff
@@ -751,7 +751,7 @@ workflows:
     description: This Workflow is to build the app using latest xcode available in Bitrise
     meta:
       bitrise.io:
-        stack: osx-xcode-16.1.x
+        stack: osx-xcode-16.0.x
```

Comment:
`osx-xcode-16.1.x` is indeed the latest stack available. We should not downgrade to 16.0. 👎🏼 
https://stacks.bitrise.io/stack_reports/osx-xcode-16.1.x/